src/app/patient-table/patient-table.component.html

```diff
@@ -1,9 +1,9 @@
-<table
-    *ngIf="!!this.dataSource"
-    mat-table
-    [dataSource]="dataSource"
-    class="mat-elevation-z8"
->
+<div class="example-container mat-elevation-z8">
```

Comment:
remove mat-elevation-z8 class please, it's making it coming out too much for my eyes

Reply:
done